DEPS

```diff
@@ -44,12 +44,14 @@ hooks = [
   {
     # Build brave-sync
     'name': 'build_brave_sync',
+    'condition': 'not checkout_android',
     'pattern': '.',
     'action': ['python', 'src/brave/script/build-simple-js-bundle.py', '--repo_dir_path', 'src/brave/components/brave_sync/extension/brave-sync'],
   },
   {
     # Build brave-crypto
     'name': 'build_brave_crypto',
+    'condition': 'not checkout_android',
```

Comment:
not sure how this is related to rewards

Reply:
It's not related to rewards. @bridiver do you remember why this change was added to android branch?

Reply:
@bridiver won't it cause problems on Android brave-core based?

Reply:
@NejcZdovc what I was missing here is that this is the merging of a lot of work on android into the main branch for browser-android-tabs so there are some changes that aren't directly related to rewards, but are necessary to make brave-core code integrate with android. It doesn't break our CI builds now for brave-core because we don't have deps on this stuff for android in brave-core yet